### PR TITLE
Adds new megaphone icon

### PR DESCRIPTION
**Pull request by XmlmXmlmX**
My megaphone icon suggestion 📣
#357

### CODE REVIEW

File: icons/megaphone.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" viewBox="0 0 16 12.375">
```
Comment (M-ZubairAhmed) [inline icons/megaphone.svg:1]: Hi @XmlmXmlmX i think BS icons have considered to take 16 16 as viewbox to be consistent with all the icons, dont you think ?

Comment (XmlmXmlmX) [reply icons/megaphone.svg:1]: true, thanks for this suggestion

Comment (M-ZubairAhmed) [reply icons/megaphone.svg]: one question, how were you able to achieve this ? did you fraction the paths to be inside of 16

Comment (XmlmXmlmX) [reply icons/megaphone.svg]: Yes, everything is inside
image

**Pull request closed without merging**